### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by kmirik999**

### CODE REVIEW

**Comment by kmirik999** [comment]
merge

File: main.py [hunk 1]
```diff
@@ -27,14 +27,25 @@ def is_game_over():
 
 
 while not is_game_over():
+    exist = False
```
Comment (tshemsedinov) [inline main.py:30]:
```suggestion
```

File: main.py [hunk 2]
```diff
@@ -27,14 +27,25 @@ def is_game_over():
...
-        guesses.append(guess)
+        for x in guesses:
+            if x == guess: exist = True
+
+        if (exist == False):
+            guessed += 1
+            guesses.append(guess)
+        else:
+            print(f"Repeat input. That word is exist.")
+            continue
```
Comment (tshemsedinov) [inline main.py:41]:
```suggestion
        if guess in guesses:
            guessed += 1
            guesses.append(guess)
        else:
            print(f"Repeat input. That word is exist.")
            continue
```

**Review by tshemsedinov — CHANGES_REQUESTED** [review]
Shame on you `if (exist == False):` equals to `if exist:`

**Pull request closed without merging**